cognite_toolkit/_cdf_tk/client/api/extended_data_modeling.py

```diff
@@ -97,9 +97,15 @@ def apply_fast(
             executor=ToolkitConcurrencySettings.get_data_modeling_write_executor(),
         )
 
-        def unwrap_element(el: T) -> InstanceApply | T:
+        def unwrap_element(el: T) -> InstanceApply | NodeId | EdgeId | T:
             if isinstance(el, dict):
-                return type(items[0])._load(el, cognite_client=self._cognite_client)
+                instance_type = el.get("instanceType")
+                if instance_type == "node":
+                    return NodeId.load(el)
+                elif instance_type == "edge":
+                    return EdgeId.load(el)
+                else:
+                    return InstanceApply._load(el)
```

Comment:
This is a fallback in case a new type of instance will be created. Not in practice it is not possible to hit this without a complaint by `mypy`.